### PR TITLE
use arrays for all.json and create legacy-all.json

### PR DESCRIPTION
**Pull request by willr3**

### CODE REVIEW

File: clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java [hunk 1]
```diff
@@ -48,6 +50,8 @@
 import io.vertx.core.spi.cluster.NodeListener;
 import io.vertx.ext.cluster.infinispan.InfinispanClusterManager;
 
+import org.infinispan.commons.api.BasicCacheContainer;
```
Comment (rvansa) [inline clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java:53]: Really?

File: clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java [hunk 2]
```diff
@@ -625,14 +629,52 @@ private void persistRun(Run run) {
             jGenerator.writeFieldName("info");
             jGenerator.writeRawValue(info.encode()); // writeObjectField() was encoding info as a POJO not json
 
-            JsonWriter.writeJson(run.statisticsStore, jGenerator, false);
+            jGenerator.writeFieldName("failures");
```
Comment (rvansa) [inline clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java:632]: Could we have this code in `JsonWriter`? Actually the whole code about creating the `all.json` would better fit there, just passing `info` as argument.

**Comment by rvansa** [comment]
@willr3 Hi, are you going to update the PR?

File: clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java (hunk 1 above)
Comment (willr3) [reply clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java:53]: What do you mean by "Really?"
The import is needed for shutdown() and I don't believe I created shutdown().

Comment (rvansa) [reply clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java:51]: Sorry, I did not notice that you've just reordered imports (I thought this was added by accident).

**Pull request merged**